lib/modulr/api/notifications_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Modulr
+  module API
+    class NotificationsService < Service
```

Comment:
Same for the Notifications namespace (aka Webhooks): resource, service, and fixtures. Specs are still missing.